src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorSubmitApplicationTest.java

```diff
@@ -361,10 +361,6 @@ void givenCaseDataWithApplicationDocumentAndServiceAuthValidationFailsInDocStore
             .andExpect(
                 status().isUnauthorized()
             );
-
-        verify(serviceTokenGenerator).generate();
-
-        verifyNoMoreInteractions(serviceTokenGenerator);
```

Comment:
this is now removed as we have scheduled tasks that run at the same time as the test runs.